src/components/network/network-table.js

```diff
@@ -598,7 +608,7 @@ const NetworkTable = (props) => {
                                     });
                                 }}
                             >
-                                <EditIcon />
+                                {!oneNodeBuilding && <EditIcon />}
```

Comment:
Maybe add check on oneNodeBuilding also on 'disable' property of parent IconButton ?

Reply:
Done